test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -360,6 +555,45 @@ public void Fluent_SetContent_Append_WritesExpectedContent()
         }
 
         [Fact]
+        public void Fluent_SetContent_AppendFormat_WritesExpectedContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = new[] { "First ", "Second Third" };
+            var i = 0;
+
+            // Act
+            tagHelperContent.SetContent("First ").AppendFormat("{0} Third", "Second");
+
+            // Assert
+            foreach (var value in tagHelperContent)
+            {
+                Assert.Equal(expected[i++], value);
+            }
```

Comment:
nit: ick doesn't `Assert.Equal()` work here?  should the two values as `IEnumerable<string>`